modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb_{context}"]
+= Using AWS Classic Load Balancers in an AWS VPC cluster extended into an Outpost
+
+Outposts racks cannot run AWS Classic Load Balancers, but Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+If you want to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you can apply labels to the edge compute nodes and configure the Classic Load Balancer to avoid scheduling on nodes with the applied labels.
+
+[NOTE]
+====
+If you do not need to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you do not need to complete these steps.
+====
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
```

Comment:
Classic Load Balancers doesn't rely on AWS Load Balancer Operator and AWS Load Balancer Controller, only AWS Application Load Balancers need to install ALB Operator and Controllers.